src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentsApi.cs

```diff
@@ -140,13 +140,13 @@ public static class EventCallback
 
         public static class EventCallbackOfT
         {
-            public static readonly string FullTypeName = "Microsoft.AspNetCore.Components.EventCallback<>";
             public static readonly string MetadataName = "Microsoft.AspNetCore.Components.EventCallback`1";
         }
 
         public static class EventCallbackFactory
         {
             public static readonly string CreateMethod = "Create";
+            public static readonly string CreateInferredMethod = "CreateInferred";
             public static readonly string CreateBinderMethod = "CreateBinder";
```

Comment:
This commit has some new APIs that I will need to add to the runtime to make this work.